torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3417,6 +3463,21 @@ def reference_lgamma(x):
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            supports_out=False,
            sample_inputs_func=sample_inputs_index_add),
+    OpInfo('__getitem__',
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           supports_out=False,
+           supports_inplace_autograd=False,
+           op=torch.Tensor.__getitem__,
+           sample_inputs_func=sample_inputs_getitem,
+           skips=(SkipInfo('TestCommon', 'test_variant_consistency_jit'),)),
```

Comment:
same comment as below

Reply:
This is the failure for this one:

<details>
<summary>click to expand</summary>

```
======================================================================
ERROR: test_variant_consistency_jit_index_put_cuda_float32 (__main__.TestCommonCUDA)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/torch/testing/_internal/common_utils.py", line 955, in wrapper
    method(*args, **kwargs)
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/torch/testing/_internal/common_utils.py", line 955, in wrapper
    method(*args, **kwargs)
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/torch/testing/_internal/common_device_type.py", line 297, in instantiated_test
    raise rte
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/torch/testing/_internal/common_device_type.py", line 292, in instantiated_test
    result = test_fn(self, *args)
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/torch/testing/_internal/common_device_type.py", line 266, in test_wrapper
    return test(*args, **kwargs)
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/test/test_ops.py", line 294, in test_variant_consistency_jit
    check_against_reference(self,
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/torch/testing/_internal/common_jit.py", line 63, in check_against_reference
    outputs_test = self.runAndSaveRNG(func, nograd_inputs, kwargs)
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/torch/testing/_internal/common_jit.py", line 130, in runAndSaveRNG
    results = func(*inputs, **kwargs)
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/torch/testing/_internal/jit_metaprogramming_utils.py", line 301, in script_fn
    fn, tensors = gen_script_fn_and_args(method_name, func_type, *args, **kwargs)
  File "/work2/kurtamohler/development/pytorch-index_put-complex-autograd/torch/testing/_internal/jit_metaprogramming_utils.py", line 292, in gen_script_fn_and_args
    CU = torch.jit.CompilationUnit(script)
RuntimeError: 
undefined value tensor:
  File "<string>", line 3

def the_method(i0, i1):
    return torch.index_put(i0, (tensor([0, 1], device='cuda:0'),), i1, accumulate=False)
                                ~~~~~~ <--- HERE

```

</details>

Reply:
no idea how to fix this off the top of my head. let's follow up on this in a separate issue/subsequent PR?

Reply:
did you get a chance to look into this?